app/main.py

```diff
@@ -1 +1,43 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.__class__.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
```

Comment:
better to use `()` then `\`
```suggestion
        return (
            f"{{Name: {self.name}, " 
            f"Health: {self.health}, " 
            f"Hidden: {self.hidden}}}"
        )
```